python-avd/pyavd/_eos_designs/structured_config/base/snmp_server.py

```diff
@@ -228,45 +194,10 @@ def _snmp_vrfs(self: AvdStructuredConfigBaseProtocol, snmp_settings: EosDesigns.
             vrf_name = self.get_vrf(vrf.name, context=f"snmp_settings.vrfs[name={vrf.name}]")
             vrfs.append_new(name=vrf_name, enable=vrf.enable)
 
-        if has_mgmt_ip and (enable_mgmt_interface_vrf := snmp_settings.enable_mgmt_interface_vrf) is not None:
-            vrfs.append_new(name=self.inputs.mgmt_interface_vrf, enable=enable_mgmt_interface_vrf)
+            if vrf.ipv4_acl is not None:
```

Comment:
so here it seems we have a change of behavior that for now I have not noticed in tests. It looks like before we were rendering IPv4 / IPv6 ACLs even if the VRF enable was not set (vrf.enable is None). Now it looks like we won't anymore because of the continue.

This may not be a problem but need to analys further